package.json

```diff
@@ -20,13 +20,19 @@
   "dependencies": {
     "axios": "^0.19.0",
     "bcrypt": "^3.0.7",
+    "dot-env": "0.0.1",
+    "dotenv": "^8.2.0",
     "express": "^4.16.3",
     "formik": "^2.0.7",
     "if-env": "^1.0.4",
     "materialize-css": "^1.0.0-rc.2",
     "mongoose": "^5.7.12",
     "node-sass": "^4.13.0",
     "sass": "^1.23.7",
+<<<<<<< HEAD
+    "sentry": "^0.1.2",
+=======
+>>>>>>> f8c32b004fe11d78a37de911d1691280e147af16
```

Comment:
need to remove the merge conflict crap

Reply:
merged and new pr requested

Reply:
also that is not needed - so if it's easier to take that out and approve this pr that works too